### ISSUE TITLE
Flawed evaluation

### ISSUE THREAD

**Issue by ieee8023** [defeatcovid19/defeatcovid19-net-pytorch#1]
> Kaggle Chest X-Ray Pneumonia dataset 

This dataset is of child X-rays. But the COVID-19 image data collection has only adults. Your model is likely just predicting age.

**Comment by mrtj** [comment]
As far as I understood, Kaggle dataset is used only to pre-train the model. The last fine tuning step is done with the https://github.com/ieee8023/covid-chestxray-dataset dataset that is of adult patients and all evaluations / metrics are calculated over a 5-fold x-validation on that dataset. So in the worst case the pretraining is not as efficient as it could be because it uses x-ray images of the chests of children instead of adults. But the final results of the model is not biased towards children/adults and there should not be any data leak because of this difference.

**Comment by thundo** [comment]
Hello @ieee8023 and thanks for your dataset! It is very useful while we try to build a COVID dataset from local hospitals.

As @mrtj noted, the two trainings are done sequentially. "Pneumonia" serves as pretrain while the final learned features are the positive/negative covid class.

Please be aware that this is intended only as template. No optimization/normalisation are in place, so the final score is not very robust.
